experimental/dds/tree2/src/test/feature-libraries/sequence-field/compose.spec.ts

```diff
@@ -1013,9 +1014,15 @@ describe("SequenceField - Compose", () => {
 			[0, 1, 2],
 			[2, 1, 0],
 		]) {
-			const move1 = tagChange(Change.move(a, 1, b > a ? b + 1 : b), tag1);
-			const move2 = tagChange(Change.move(b, 1, c > b ? c + 1 : c), tag2);
-			const return2 = tagRollbackInverse(
+			const move1: TaggedChange<SF.Changeset<never>> = tagChange(
```

Comment:
Not sure why I had to add these type annotations. After my changes I was suddenly getting an error here and had to explicitly specify the type. You'll see this change in a good number of files.